src/WalkingModule/app/robots/iCubGazeboV2_5/dcm_walking/common/plannerParams.ini

```diff
@@ -1,5 +1,5 @@
 ##Timings
-plannerHorizon          10.0
+plannerHorizon          5.0
```

Comment:
Thanks. After the separation of the folder, How we can use it inside the code in the situation that step adjustment is active?

Reply:
I added a separate folder with the separate main config file for step adjustment and now how we can specify when should we use this file?

Reply:
A possibility could be to add another section, similar to https://github.com/MiladShafiee/walking-controllers/blob/1e3756ec09c5fd04c3aaa9cc891ce99cb03faff2/src/WalkingModule/app/robots/iCubGazeboV2_5/dcm_walking_with_joypad.ini#L33.
For example, you can add a line like
```ini
[include TRAJECTORY_PLANNER_PLUS_ADAPTATION "/path/to/the/modifed/file"]
```
then inside the code you decide which section to load. The problem here is that it may be difficult to understand which file is loaded and there may be incompatibilities with other demos.
Another possibility would be to treat it as a different demo. For example, see the difference between these two files:
- https://github.com/MiladShafiee/walking-controllers/blob/1e3756ec09c5fd04c3aaa9cc891ce99cb03faff2/src/WalkingModule/app/robots/iCubGazeboV2_5/dcm_walking_retargeting.ini
- https://github.com/MiladShafiee/walking-controllers/blob/1e3756ec09c5fd04c3aaa9cc891ce99cb03faff2/src/WalkingModule/app/robots/iCubGazeboV2_5/dcm_walking_with_joypad.ini

The problem, in this case, would be that it is difficult to merge the different demos together. It would not be possible to do retargeting with step adaptation as it is, but it would be necessary to add yet another folder. I would ask also @GiulioRomualdi opinion on this.

Reply:
Thanks! In the previous modification I used the second option. I'm just wondering, in the second case for running different demos where exactly(inside the code) we can specify which demo should be run?

Reply:
It depends on how you run the module. By default, the module looks for the ``dcm_walking_with_joypad.ini`` file because of this line: https://github.com/MiladShafiee/walking-controllers/blob/9175a1f55425b62d3b794ec62c3a0916fb9147cf/src/WalkingModule/src/main.cpp#L31
but if you call the module like
```
WalkingModule --from dcm_walking_retargeting.ini
```
it will look for the ini file of the retargeting.

Reply:
I got it. Thank you!

Reply:
@GiulioRomualdi @S-Dafarra  Here I used the second proposal that is option of having different demo. What do you think?

Reply:
Good for me. I think you are not calling the correct robotControl.ini from the main configuration file though 🤔

Reply:
I checked it but I couldn't find your point! Which file are you talking about?  iCubGazebo or iCubGenova?

Reply:
I meant this line https://github.com/MiladShafiee/walking-controllers/blob/c633353d6f7156e4cbdeea25b96ffae3126ed56d/src/WalkingModule/app/robots/iCubGazeboV2_5/dcm_walking_step_adaptation.ini#L30

Shouldn't it call the corresponding ``roboControl.ini`` in the ``step_adaptation`` folder?

Reply:
Thanks! There was a mistake between `robotControl.ini `and `ControlParameter.ini`!

Reply:
Done!